lib/api/v3/work_packages/work_package_collection_representer.rb

```diff
@@ -39,10 +41,25 @@ def initialize(models,
                        project: nil,
                        page: nil,
                        per_page: nil,
-                       embed_schemas: false)
+                       embed_schemas: false,
+                       timestamps: [],
+                       _query: nil)
```

Comment:
When implementing this, I've thought about maybe leaving out the `timestamps` parameter here, because it is also available through the `query`.  But I liked the explicit mention of the `timestamps`. Also, it made the specs slightly nicer.